app/views/reports/_form.html.erb

```diff
@@ -1,11 +1,11 @@
 <%= turbo_frame_tag "report_form" do %>
     <%= form_with scope: :report, url: report_path, method: :put, data: { controller: :form } do |f| %>
       <div class="flex flex-col gap-5">
-        <div class="grid gap-3 grid-cols-2">
-          <h2 class="text-xl">
+        <div class="grid gap-8 grid-cols-2">
+          <h2 class="text-xl ">
```

Comment:
**Nitpicky**: this space should not be part of the PR. 

Please review your own PR's before sending them over to me :smile: